src/plugins/irc-events/connection.js

```diff
@@ -63,23 +62,15 @@ module.exports = function(irc, network) {
 		}), true);
 	});
 
-	if (identd.isEnabled()) {
-		irc.on("raw socket connected", function(socket) {
-			identd.hook(socket, client.name || network.username);
-		});
-	}
+	let identSocketId;
 
-	if (identHandler) {
-		let identSocketId;
-
-		irc.on("raw socket connected", function(socket) {
-			identSocketId = identHandler.addSocket(socket, client.name || network.username);
-		});
+	irc.on("raw socket connected", function(socket) {
```

Comment:
Was there a discussion on the IRC channel with prawnsalad saying we should not use `raw` event? Very likely I am wrong on this, but just double-checking.